[debug] [2019-05-02T07:35:24.903Z] ----------------------------------------------------------------------
[debug] [2019-05-02T07:35:24.905Z] Command:       /Users/clarabrandt/.nvm/versions/node/v10.11.0/bin/node /Users/clarabrandt/.nvm/versions/node/v10.11.0/bin/firebase deploy
[debug] [2019-05-02T07:35:24.905Z] CLI Version:   6.4.0
[debug] [2019-05-02T07:35:24.905Z] Platform:      darwin
[debug] [2019-05-02T07:35:24.905Z] Node Version:  v10.11.0
[debug] [2019-05-02T07:35:24.906Z] Time:          Thu May 02 2019 09:35:24 GMT+0200 (Central European Summer Time)
[debug] [2019-05-02T07:35:24.906Z] ----------------------------------------------------------------------
[debug] 
[debug] [2019-05-02T07:35:24.915Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2019-05-02T07:35:24.915Z] > authorizing via signed-in user
[debug] [2019-05-02T07:35:24.915Z] [iam] checking project pigeon-90548 for permissions ["firebase.projects.get","firebasedatabase.instances.update","firebasehosting.sites.update"]
[debug] [2019-05-02T07:35:24.917Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/pigeon-90548:testIamPermissions  

[debug] [2019-05-02T07:35:25.894Z] <<< HTTP RESPONSE 200
[debug] [2019-05-02T07:35:25.897Z] >>> HTTP REQUEST GET https://firebase.googleapis.com/v1beta1/projects/pigeon-90548  
 
[debug] [2019-05-02T07:35:26.398Z] <<< HTTP RESPONSE 200
[info] 
[info] === Deploying to 'pigeon-90548'...
[info] 
[info] i  deploying database, hosting
[debug] [2019-05-02T07:35:26.406Z] Error: ENOENT: no such file or directory, open '/Users/clarabrandt/workspace/pigeon_project/ccb-pigeons/database.rules.json'
    at Object.openSync (fs.js:436:3)
    at Object.readFileSync (fs.js:341:35)
    at Config.readProjectFile (/Users/clarabrandt/.nvm/versions/node/v10.11.0/lib/node_modules/firebase-tools/lib/config.js:143:26)
    at /Users/clarabrandt/.nvm/versions/node/v10.11.0/lib/node_modules/firebase-tools/lib/deploy/database/prepare.js:42:50
    at /Users/clarabrandt/.nvm/versions/node/v10.11.0/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:4911:15
    at baseForOwn (/Users/clarabrandt/.nvm/versions/node/v10.11.0/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:2996:24)
    at /Users/clarabrandt/.nvm/versions/node/v10.11.0/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:4880:18
    at Function.forEach (/Users/clarabrandt/.nvm/versions/node/v10.11.0/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9344:14)
    at module.exports (/Users/clarabrandt/.nvm/versions/node/v10.11.0/lib/node_modules/firebase-tools/lib/deploy/database/prepare.js:39:7)
    at _chain (/Users/clarabrandt/.nvm/versions/node/v10.11.0/lib/node_modules/firebase-tools/lib/deploy/index.js:22:40)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-05-02T07:35:40.882Z] ----------------------------------------------------------------------
[debug] [2019-05-02T07:35:40.883Z] Command:       /Users/clarabrandt/.nvm/versions/node/v10.11.0/bin/node /Users/clarabrandt/.nvm/versions/node/v10.11.0/bin/firebase deploy
[debug] [2019-05-02T07:35:40.884Z] CLI Version:   6.4.0
[debug] [2019-05-02T07:35:40.884Z] Platform:      darwin
[debug] [2019-05-02T07:35:40.884Z] Node Version:  v10.11.0
[debug] [2019-05-02T07:35:40.884Z] Time:          Thu May 02 2019 09:35:40 GMT+0200 (Central European Summer Time)
[debug] [2019-05-02T07:35:40.884Z] ----------------------------------------------------------------------
[debug] 
[debug] [2019-05-02T07:35:40.893Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2019-05-02T07:35:40.894Z] > authorizing via signed-in user
[debug] [2019-05-02T07:35:40.894Z] [iam] checking project pigeon-90548 for permissions ["firebase.projects.get","firebasedatabase.instances.update","firebasehosting.sites.update"]
[debug] [2019-05-02T07:35:40.895Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/pigeon-90548:testIamPermissions  

[debug] [2019-05-02T07:35:41.678Z] <<< HTTP RESPONSE 200
[debug] [2019-05-02T07:35:41.682Z] >>> HTTP REQUEST GET https://firebase.googleapis.com/v1beta1/projects/pigeon-90548  
 
[debug] [2019-05-02T07:35:41.900Z] <<< HTTP RESPONSE 200
[info] 
[info] === Deploying to 'pigeon-90548'...
[info] 
[info] i  deploying database, hosting
[debug] [2019-05-02T07:35:41.907Z] Error: ENOENT: no such file or directory, open '/Users/clarabrandt/workspace/pigeon_project/ccb-pigeons/database.rules.json'
    at Object.openSync (fs.js:436:3)
    at Object.readFileSync (fs.js:341:35)
    at Config.readProjectFile (/Users/clarabrandt/.nvm/versions/node/v10.11.0/lib/node_modules/firebase-tools/lib/config.js:143:26)
    at /Users/clarabrandt/.nvm/versions/node/v10.11.0/lib/node_modules/firebase-tools/lib/deploy/database/prepare.js:42:50
    at /Users/clarabrandt/.nvm/versions/node/v10.11.0/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:4911:15
    at baseForOwn (/Users/clarabrandt/.nvm/versions/node/v10.11.0/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:2996:24)
    at /Users/clarabrandt/.nvm/versions/node/v10.11.0/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:4880:18
    at Function.forEach (/Users/clarabrandt/.nvm/versions/node/v10.11.0/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9344:14)
    at module.exports (/Users/clarabrandt/.nvm/versions/node/v10.11.0/lib/node_modules/firebase-tools/lib/deploy/database/prepare.js:39:7)
    at _chain (/Users/clarabrandt/.nvm/versions/node/v10.11.0/lib/node_modules/firebase-tools/lib/deploy/index.js:22:40)
[error] 
[error] Error: An unexpected error has occurred.
